using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -434,6 +670,9 @@ Use this API to send a prefetch request to your configured Target server with th
 withParameters: (nullable ACPTargetParameters*) targetParameters
 callback: (nullable void (^) (NSError* _Nullable error)) callback;
 ```
+* _targetPrefetchObjectArray_ is an array of `ACPTargetPrefetchObject` objects for various mbox locations.
+* _targetParameters_ is the configured `ACPTargetParameters` for the prefetch request.
+* _callback_ is the callback invoked with a `nil` value if prefetch is successful, otherwise an error message is returned.
 
```

Comment:
Please reword the bulleted list as follows:
"* *targetPrefetchObjectArray* is an array of `ACPTargetPrefetchObject` objects for various mbox locations.
* *targetParameters* is the configured `ACPTargetParameters` for the prefetch request.
* If the prefetch is successful, *callback* is invoked with a `nil` value. If the prefetch is not successful, an error message is returned."